routes/tasks.go

```diff
@@ -0,0 +1,142 @@
+package routes
+
+import (
+	_ "github.com/go-sql-driver/mysql"
+	"github.com/labstack/echo"
+	"net/http"
+	"time"
+
+	"github.com/kirikami/go_exercise_api/database"
+	u "github.com/kirikami/go_exercise_api/utils"
+)
+
+func (h ApiV1Handler) SaveTaskHandler(c echo.Context) error {
+	task := database.Task{}
+
+	db := h.DB
+
+	err := c.Bind(&task)
+
+	if err != nil {
+		return err
+	}
+
+	err = db.Save(&task).Error
+	if err != nil {
+		return err
+	}
+
+	return c.JSON(http.StatusCreated, task)
+
+}
+
+func (h ApiV1Handler) UpdateTaskHandler(c echo.Context) error {
+	currentTime := time.Now()
+	db := h.DB
+	idParam := c.P(0)
+
+	if idParam == "" {
+		return echo.NewHTTPError(http.StatusNotFound)
+	}
+
+	id, err := u.ParseIdInt64FromString(idParam)
+
+	if err != nil {
+		return err
+	}
+
+	task := database.Task{}
+	err = db.First(&task, id).Error
+
+	if err != nil {
+		return err
+	}
+
+	err = c.Bind(&task)
```

Comment:
Steps should be
1) Try to find task by ID, if there is no task - return an error `not found`
2) Bind request, if request is not valid - return bad request error
3) Update changes